### ISSUE TITLE
accessibility issue 29 (medium) -tab focus and keyboard enter action on thumbnail images

### ISSUE THREAD

**Issue by yulgit1** [ycba-cia/blacklight-collections2#246]
The focus is not visible for the thumbnail images at the bottom of the larger image (these thumbnail images are used to select the image to view).

**Comment by yulgit1** [comment]
resolved:https://github.com/ycba-cia/blacklight-collections2/commit/1aeb00c67a85a390b9fc7c95704b6a6cd74a49bc
